app/forms/concept_form.rb

```diff
@@ -0,0 +1,13 @@
+class ConceptForm
```

Comment:
@VivianChu What design pattern it belongs to? I think it might not be the design pattern but just the way it come up for the forms. DHH suggested Fat model, skinny controller. And I understand that in wild apps, people broke that rule for various reasons.